openhtf/io/output/console_summary.py

```diff
@@ -0,0 +1,107 @@
+"""Module to display test summary on console."""
+
+import os
+import sys
+import __main__ as main
```

Comment:
i'd rather use the tester name for this, the main entry point name is commonly meaningful, but not necessarily, and we have an explicit tester naming mechanism, so we might as well use it.